UI/index.html

```diff
@@ -0,0 +1,434 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <link href="styles.css" rel="stylesheet" charset="utf-8">
+    <meta charset="utf-8">
+    <title>SocNet</title>
+  </head>
+  <body>
```

Comment:
Try to reset body's margins to avoid empty spaces under the footer and above header